python/src/aiconfig/editor/client/src/components/AIConfigEditor.tsx

```diff
@@ -940,6 +940,20 @@ export default function AIConfigEditor({
           <Container maw="80rem">
             <Flex justify="flex-end" mt="md" mb="xs">
               <Group>
+                {/* TODO: Remove false gating below once Share button is ready */}
+                {false && !readOnly && (
```

Comment:
Let's also make this predicated on a 'getShareLink' callback existing so that we only show this in gradio (where we can pass through the function implementation that calls gradio backend)

Reply:
Can be done in later PR

Reply:
> so that we only show this in gradio
I was thinking of just using MODE, but yea this works too

> Let's also make this predicated on a 'getShareLink' callback existing 
Yea I have this planned for a future PR